src/VirtoCommerce.ContentModule.Web/Scripts/blades/pages/edit-blog.js

```diff
@@ -31,7 +30,12 @@ angular.module('virtoCommerce.contentModule')
                     }
                     return !(blade.existedBlogsName && blade.existedBlogsName.includes(value));
                 };
-                
+
+                $scope.validateName = (value) => {
+                    var pattern = /^[a-z0-9_.-]+$/;
```

Comment:
I think we also should change the validation message here, as uppercase letters are not special symbols:
https://github.com/VirtoCommerce/vc-module-content/blob/e190682e156b246425654be8df390f92e16b9a39/src/VirtoCommerce.ContentModule.Web/Validators/ContentFolderValidator.cs#L27
Also please align client validation with the symbol set from the rule: 
[^0-9a-z -]